.github/workflows/raspberrypi3-64.yml

```
@@ -0,0 +1,47 @@
+name: Raspberry Pi 3 (using 64bit OS)
+
+on:
+  # With these triggers the Yocto jobs will run
+  # in parallel with the Flowzone jobs, which is fine for now
+  # and allows us to better control what we want to test and when.
+  # It is expected that Flowzone could fail, but yocto jobs will run.
+  pull_request:
+    branches:
+      - "main"
+      - "master"
+  pull_request_target:
+    branches:
+      - "main"
+      - "master"
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+\+?r?e?v?*
+      - v20[0-9][0-9].[0-1]?[1470].[0-9]+
+  workflow_dispatch:
+
+jobs:
+  yocto:
+    name: Yocto
+    # FIXME: This workflow has dependencies on scripts in the balena-yocto-scripts repository
+    # which is pinned separately as a submodule in the device repo. Expect some drift but try to retain compatibility.
+    uses: balena-os/balena-yocto-scripts/.github/workflows/yocto-build-deploy.yml@d59fac4cce1dcff0b423ac97aeccbd7f4486b9c2 # v1.25.8
+    # Prevent duplicate workflow executions for pull_request (PR) and pull_request_target (PRT) events.
+    # Both PR and PRT will be triggered for the same pull request, whether it is internal or from a fork.
+    # This condition will prevent the workflow from running twice for the same pull request while
+    # still allowing it to run for all other event types.
+    if: (github.event.pull_request.head.repo.full_name == github.repository) == (github.event_name == 'pull_request')
+    secrets: inherit
+    with:
+      machine: raspberrypi3-64
+      # Don't deploy any artifacts for RPI build+test sanity workflows
+      deploy-s3: false
+      deploy-hostapp: false
+      deploy-ami: false
+      # worker_type defaults to testbot
+      # worker_fleets defaults to balena/testbot-rig,balena/testbot-rig-partners,balena/testbot-rig-x86,balena/testbot-rig-partners-x86
+      test_matrix: >
+        {
+          "test_suite": ["os","cloud","hup"],
+          "environment": ["bm.balena-dev.com"],
+          "runs_on": [["ubuntu-20.04"]]
```

```suggestion
          "runs_on": [["ubuntu-latest"]]
```
The 20.04 runners are going to be EOL soon, might as well go with 22.04 or latest since it shouldn't impact the client.

---

latest be better